xarrayutils/utils.py

```diff
@@ -887,3 +887,40 @@ def concat_dim_da(data, name):
     return xr.DataArray(data, dims=[name],
                         coords={name: (name, data)},
                         name=name)
+
+
+
+def da_detrend(a, b, start_shift=0, timedim='time'):
```

Comment:
Can we refactor `timedim` into `dim` throughout the function? Its more general and more in line with the xarray syntax.

Reply:
sure. maybe xdim?